src/App.elm

```diff
@@ -68,11 +68,17 @@ update msg model =
         FetchMeLoaded result ->
             case result of
                 Ok me ->
-                    ( { model
-                        | loggedIn = LIUpdate.setMe model.loggedIn me
-                      }
-                    , Cmd.none
-                    )
+                    case model.authStatus of
+                        LoggedIn loginInfo ->
+                            ( { model
+                                | loggedIn = LIUpdate.setMe model.loggedIn me
+                              }
+                            , fetchConversations loginInfo me.id
+                                |> Task.perform FetchConversations FetchConversations
+                            )
+
+                        LoggedOut ->
+                            ( model, Cmd.none )
```

Comment:
actually there's also a `Result.withDefault` - want to use it here too?

```elm
result
	|> Result.map
		(\me ->
			case model.authStatus of
				…
		)
	|> Result.withDefault
```

if not, that's okay too